_projects/2024/100542532/100542532.Rmd

```diff
@@ -0,0 +1,532 @@
+
+```{r}
+font_add_google(name = "Roboto Condensed", family = "Roboto Condensed Regular", regular.wt = 400)
+font_add_google(name = "Roboto Condensed", family = "Roboto Condensed Bold", regular.wt = 700)
+font_add_google(name = "Roboto Condensed", family = "Roboto Condensed Light", regular.wt = 300)
+showtext_auto()
+```
+
+I customized my graph's theme by removing unnecessary elements like x-axis text and grid lines. As I mentioned earlier, I also adjusted the y-axis text for size and alignment. I then specified specific font weights, sizes, and spacing for the title, subtitle, and caption. Finally, I positioned the legend at the top in a horizontal layout and fine-tuned the margins to closely replicate the overall design and layout.
+
+```{r}
+my_graph <- my_graph +
+  theme_minimal(base_family = "Roboto Condensed Regular") +  
+  theme(
+    axis.text.x = element_blank(),  
+    axis.ticks.x = element_blank(), 
+    panel.grid.major.x = element_blank(), 
+    panel.grid.minor = element_blank(),   
+    axis.text.y.right = element_text(size = 7, vjust = -0.5, hjust = -5.5), 
+    plot.title = element_text(size = 9.5, family = "Roboto Condensed Bold", margin = margin(b = 3)), 
+    plot.subtitle = element_text(size = 8.5, family = "Roboto Condensed Light", margin = margin(b = 10)), 
+    plot.caption = element_text(hjust = 0, size = 6, color = "#808080", margin = margin(b = -10)), 
+    plot.caption.position = "plot", 
+    plot.margin = margin(t = 17, r = -5, b = 10, l= 10), 
+    legend.position = "top", 
+    legend.direction = "horizontal", 
+    legend.justification = "left", 
+    legend.margin = margin(-5, 0, 0, 0), 
+    legend.text = element_text(size = 7), 
+    legend.key.size = unit(0, 'cm') 
+  )
+my_graph
+```
+
+### **Annotations**
+
+The last step was to add in the annotations using `annotate().` I started by adding the annotations for the five selected cities in the original graph.
+
+```{r}
+my_graph <- my_graph +
+ annotate("text", x = which(data$City == "New York, NY"), y = 144, label = "New York, NY", 
+           size = 2.2, hjust = 0, family = "Roboto Condensed Regular") +
+  annotate("text", x = which(data$City == "Miami, FL"), y = 94, label = "Miami, FL", 
+           size = 2.2, hjust = 0, family = "Roboto Condensed Regular") +
+  annotate("text", x = which(data$City == "Charleston, SC"), y = 38, label = "Charleston, SC", 
+           size = 2.2, hjust = 0.9, family = "Roboto Condensed Regular") +
+  annotate("text", x = which(data$City == "Seattle, WA"), y = 66, label = "Seattle, WA", 
+           size = 2.2, hjust = 0, family = "Roboto Condensed Regular") +
+  annotate("text", x = which(data$City == "El Paso, TX"), y = 28, label = "El Paso, TX", 
+           size = 2.2, hjust = 0.95, family = "Roboto Condensed Regular") 
+my_graph
+```
+
+Finally, I added the fork structure connecting Los Angeles and Chicago, as well as the annotation text to complete my static replication.
+
+```{r, out.width='100%'}
```

Comment:
Please add `preview = TRUE`, so that this image is shown in the list of projects.